tests/helpers.test.js

```diff
@@ -0,0 +1,511 @@
+const _ = require("lodash");
+const consts = require("../consts.json");
+const helpers = require("../helpers");
+const rewire = require("rewire");
+
+describe("removeCredentials", () => {
+  test("properly removes credentials from params based on DEFAULT_CREDENTIAL_LABELS if no labels provided", () => {
```

Comment:
`DEFAULT_CREDENTIAL_LABELS` is an implementation detail, we don't need to know about it here.
`Removes credentials from params`?